phpunit.xml.dist

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
PHPUnit checks if _phpunit.xml.dist_ exists after checking if _phpunit.xml_ exists.

This allows you to have an standard XML (_phpunit.xml.dist_), but also that you can have a custom XML (_phpunit.xml_).